SwrveSDK/src/main/java/com/swrve/sdk/messaging/SwrveCampaignState.java

```diff
@@ -37,7 +37,7 @@ public static Status parse(String status) {
     }
 
     // Number of impressions of this campaign. Used to disable the campaign if it reaches total impressions
-    protected int impressions;
+    public int impressions;
```

Comment:
I made it public because the Displayer needed access to it. However, I'll make it protected again and use a getter for this.